packages/tools/fetch-tool/src/fluidFetchSnapshot.ts

```diff
@@ -37,15 +37,15 @@ interface IFetchedBlob {
     treePath: string;
     filename: string;
```

Comment:
Can you please run fluid-fetch tool once to ensure it works? I.e. pick any file and download it with --saveDir <> --stat options?

Reply:
great catch! there was an error. so I added a line